### ISSUE TITLE
"Attend virtually" is below the fold on `/attend` on a laptop screen

### ISSUE THREAD

**Issue by strugee** [SeaGL/seagl.github.io#441]
I worry that people will get to this page and _literally not know to scroll down_ in order to see the "attend virtually" options:

![image](https://github.com/user-attachments/assets/28c49f2a-f86e-4c72-8831-2a70c594dff2)

**Comment by altsalt** [comment]
Not as concerned because we'll have the day-of ribbon... Also we should find the spot where we turn that ribbon on...

**Comment by strugee** [comment]
Yeah, but I want to advertise ahead of time without having to write "(scroll down for virtual conference instructions)". I guess it's not the end of the world though, yes.
